contrib/endpoints/src/api_manager/context/request_context.h

```diff
@@ -112,6 +112,16 @@ class RequestContext {
     last_report_time_ = tp;
   }
 
+  void set_auth_claims(std::string &claims) {
+    auth_claims_ = claims;
+  }
+
+  std::string &auth_claims() {
```

Comment:
Why? We just use this string to create a proto object and this string can be potentially big.